statsmodels/regression/tests/test_glsar_gretl.py

```diff
@@ -368,7 +370,7 @@ def test_all(self):
         #res2 = res.get_robustcov_results(cov_type='HC1')
         # TODO: fvalue differs from Gretl, trying any of the HCx
         #assert_almost_equal(res2.fvalue, result_gretl_g1['fvalue'][1], decimal=0) #FAIL
-        #assert_approx_equal(res.f_pvalue, result_gretl_g1['f_pvalue'][1], significant=1) #FAIL
+        #assert_array_less(res.f_pvalue, result_gretl_g1['f_pvalue'][1], rtol=1e-1) #FAIL
```

Comment:
don't change comments like that, the new version wouldn't be identical and so the comment might not be correct anymore

besides using assert_array_less is the wrong assert anyway.

Reply:
Huh, no idea what past-me had in mind here.  Will fix.

Reply:
Just pushed with fix.